src/Agent/NewRelic/Agent/Core/Config/Configuration.xsd

```diff
@@ -2072,6 +2072,23 @@
                         </xs:all>
                     </xs:complexType>
                 </xs:element>
+
+                <xs:element name="openTelemetryBridge" minOccurs="0" maxOccurs="1">
```

Comment:
Can we avoid making this change in the xsd file? We still need to figure out whether we want customers to be able to turn this off once the feature is GA. For that reason, we should treat this setting as a feature flag, and define an appsetting for it.